tasks/loggly.yml

```diff
@@ -0,0 +1,39 @@
+- name: Create cerfiticates folder if it does not exist
```

Comment:
I'm assuming these are moved from `main.yml` with formatting changes, right (i.e. no functionality changes in this file)?